scripts/ci-pull-request-closed.sh

```diff
@@ -21,33 +21,17 @@ if [[ "$GITHUB_EVENT_NAME" == "pull_request" && ! -z "$GITHUB_EVENT_PATH" ]]; th
     if [[ "$pr_action" == "closed" ]]; then
         pr_comment_api_url="$(echo $event | jq -r ".pull_request._links.comments.href")"
 
-        # Find all commits associated with the PR.
-        pr_commits="$(curl \
-            -s \
-            -H "Authorization: token ${GITHUB_TOKEN}" \
-            -H "Accept: application/vnd.github.v3+json" \
-            "https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/${pr_number}/commits")"
-
-        # For each PR commit, if a bucket exists for it, delete it.
-        for commit in $(echo $pr_commits | jq -r ".[].sha"); do
-            pr_bucket_name="$(get_bucket_for_commit $commit)"
-
-            if [ ! -z "$pr_bucket_name" ]; then
-                echo "Found bucket ${pr_bucket_name} associated with commit ${commit}."
-                echo "Attempting to delete..."
-
-                # aws s3api head-bucket is a quick way to determine whether the bucket exists
-                # and we have access to it.
-                if aws s3api head-bucket --bucket "$pr_bucket_name" --region "$(aws_region)" 2>/dev/null; then
-                    aws s3 rb "s3://${pr_bucket_name}" --force --region "$(aws_region)"
-                    remove_param_for_commit "$(git_sha)" "$(aws_region)"
```

Comment:
Does this mean we no longer need SSM params to map these? Should we include the change to remove setting the params if we're going to stop deleting (or trying to delete) them?

Reply:
Yeah, if it's truly dead code, we should remove it.

That said, I'm still curious as to what actually stopped working here, as we went for so long time without any problems with it. Were you able to figure that out? (What you have here looks fine on the surface -- I'm just _fairly_ certain I did things like this for a reason, though admittedly it's been so long I no longer remember what it was.)

Reply:
Yeah I removed it. They were failing because the call to the ssm parameter store was failing for some odd reason. I then figured I would just remove it and simplify things since it doesn't look like this really adds too much value.

I figure this was done because the state of the world was much different at the time this was written. All the previews were created in the ci account and multiple repos were writing to it. So maybe extra care was taken to avoid the chance for collision if you were to only use PR numbers. Currently we are operating in the testing-website account and this and the private repo are the only two repos writing to it. If for some reason we add other repos down the road that are standing up buckets here, the bucket naming scheme will end up being different, so there is no chance for collision. The only potential here is PR numbers colliding with the private repo, but the PR number count in the private repo is still under 100 and here we are over 12K. So the numbers will _never_ collide because it is so far behind and we rarely use the repo by comparison. Anyways, I just do not see the value for the added complexity here.

Reply:
>extra care was taken to avoid the chance for collision if you were to only use PR numbers

Yep, I believe that was it -- PR numbers weren't specific enough. Ok, seems reasonable enough. Thanks for jogging my memory!